static/app/components/events/interfaces/crashContent/exception/sourceMapDebug.tsx

```diff
@@ -153,16 +154,9 @@ function getErrorMessage(
           docsLink: getTroubleshootingLink(),
         },
       ];
-    case SourceMapProcessingIssueType.NOT_PART_OF_PIPELINE:
-      return [
-        {
-          title: t('Sentry not part of build pipeline'),
-          desc: t(
-            'Integrate Sentry into your build pipeline using a tool like Webpack or the CLI.'
-          ),
-          docsLink: defaultDocsLink,
-        },
-      ];
+    // Need to return something but this does not need to follow the pattern since it uses a different alert
+    case SourceMapProcessingIssueType.DEBUG_ID_NO_SOURCEMAPS:
+      return [{title: 'Debug Id but no Sourcemaps'}];
```

Comment:
@roggenkemper what about the description and the docs link? This error message doesn't seem super helpful.

Reply:
This error message isn't actually shown anywhere , since we are just showing the source maps wizard banner, the description/docs link are irrelevant, so I just wanted to return something (to distinguish it from the default return).